sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.queryPlan

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Is this file required?

Reply:
'query plan' is empty contents file.
another test is the same this too.

(please refer to https://github.com/CUBRID/cubrid-testcases/blob/develop/sql/_14_mysql_compatibility_2/_15_host_variable/_12_common/cases/where_clause.queryPlan)